examples/sample_pipeline/pipeline.py

```diff
@@ -50,13 +50,17 @@
     ],
 )
 class CalculateChunkLength(PandasTransformComponent):
+    def __init__(self, feature_name: str, **kwargs):
+        self.feature_name = feature_name
+
     def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
-        dataframe["chunk_length"] = dataframe["text"].apply(len)
+        dataframe[self.feature_name] = dataframe["text"].apply(len)
         return dataframe
 
 
 _ = dataset.apply(
     ref=CalculateChunkLength,
     consumes={"text": pa.string()},
     produces={"chunk_length": pa.int32()},
+    arguments={"feature_name": "chunk_length"},
```

Comment:
This is a bit confusing, can't we just keep it as it was before and pass a random string argument